src/libPMacc/include/functor/Filtered.hpp

```diff
@@ -110,8 +109,10 @@ namespace acc
 
     /** specialization of Filtered (with unary filter)
      *
-     * This specialization can only used if T_Filter is of the type PMacc::filter::Interface
-     * and T_Functor is of the type PMacc::filter::Interface
+     * This specialization can only be used if T_Filter is of the type PMacc::filter::Interface
+     * and T_Functor is of the type PMacc::functor::Interface.
+     * An unary filters means that each argument can only pass the same filter
```

Comment:
A~~n~~ unary filter~~s~~

Furthermore I don't understand what you want to tell me with the sentence.